packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md

```diff
@@ -1 +1,7 @@
-This functionality is coming soon. Watch this GitHub issue for updates: [#126 Support OIDC logout](https://github.com/okta/okta-spring-boot/issues/126)
+You can configure an RP-Initiated (SSO) Logout by setting the `okta.oauth2.postLogoutRedirectUri` property with an absolute URI such as:
+
+```properties
+okta.oauth2.postLogoutRedirectUri=http://localhost:8080/
```

Comment:
```suggestion
okta.oauth2.post-logout-redirect-uri=http://localhost:8080/
```

As of the current major version of Spring, kabob case is recommended.

Reply:
Actually, i just tried this, and it didn't work.  I'm 99% sure the fix is trivial:
https://github.com/okta/okta-spring-boot/issues/175

I try to make time to try that later today

Reply:
Will address okta/okta-spring-boot#175 in a separate PR (outside the scope of this PR) to keep this PR unblocked.